styles.css

```diff
@@ -429,3 +433,16 @@ a.mod_kanban_attachment_item {
     border-top-right-radius: 10px;
     border-bottom-right-radius: 10px;
 }
+
+.mod_kanban_number {
+    font-weight: bold;
+}
+
+.mod_kanban_card_number {
+    cursor: pointer;
+    font-weight: bold;
```

Comment:
Ich fände es besser, wenn die cardnumber in der description anders gestylt ist, als die Number im Titel. Ggf. könnte man die, in der description, ähnlich wie einen Link darstellen.